src/components/Sidebar/Contacts/Contacts.js

```diff
@@ -7,16 +7,18 @@ const Contacts = ({ contacts }) => (
   <div className={styles['contacts']}>
     <ul className={styles['contacts__list']}>
       {Object.keys(contacts).map((name) => (
```

Comment:
I think it's better to check if the contact is empty and also use filter

```{Object.keys(contacts).filter((name) => contacts[name] !== '').map((name) => (```

Reply:
I'm totally fine with that.  Probably wouldn't be a bad idea if the Contacts component didn't mount if the contacts array is empty.  But that is above my level of expertise at the moment.